README.md

```diff
@@ -301,6 +301,25 @@ Application.put_env(:wallaby, :js_logger, file)
 
 Logging can be disabled by setting `:js_logger` to `nil`.
 
+## Config
+
+### Adjusting timeouts
+
+Wallaby uses [hackney](https://github.com/benoitc/hackney) under the hood, so we
```

Comment:
technically it uses httpoison which uses hackney, should be fine though - was just a bit confusing to me.